CHANGELOG.adoc

```diff
@@ -10,11 +10,23 @@ and this project adheres to https://semver.org/spec/v2.0.0.html[Semantic Version
 
 == Unreleased
 
+## [1.0.0] - 2022-10-07
+### Added
+
 === icon:rocket[] Added
 
 * Introduce a changelog (#30)
 * Added support for using the JavaScript libraries offline (#25)
+* Add captioned title on charts (#18)
+* Improved documentation
+* Added attributes to define chart resources directory (#26)
+* Introduce visual regression tests (#34)
 
 === icon:heartbeat[] Changed
```

Comment:
we should include https://github.com/asciidoctor/asciidoctor-chart/pull/17

Reply:
Split it between Changed and Fixed cause it was both bugfixing and general changes